packages/frontend/src/components/Banner/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { useBannerStore } from '../../store/bannerStore';
+import { Alert, Collapse, IconButton } from '@mui/material';
+import { Colors } from '../../constants/Colors';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import InfoIcon from '@mui/icons-material/Info';
+import './styles.css';
+
+export const Banner = () => {
+  const banners = useBannerStore((state) => state.banners);
+  const getBanners = useBannerStore((state) => state.getBanners);
+
+  const [open, setOpen] = useState(true);
+
+  useEffect(() => {
+    getBanners();
+  }, [getBanners]);
+
+  return (
+    <>
+      <div style={{ margin: '20px 32px 0 32px' }}>
+        <IconButton onClick={() => setOpen(!open)} className="banner-alert">
```

Comment:
I don't think users will realize that you can hide the banner with the Info icon, but we can tweak these small things later.